### ISSUE TITLE
Suggestion for checker to travel subfolders

### ISSUE THREAD

**Issue by trevor-cockrell** [mlcommons/power-dev#173]
This issue is not urgent and can be taken as suggestion for future rounds.

A problem I ran into was that I was having to send over a parent folder instead of the direct loadgen folder. 
This was because of the exact timestamp applied to the output logs, which is not feasible to pass to the `client` call for the `-L` logs flag. 

This is what was being used by the checker:
```
run_1 / ranging
└── 2021.03.02-09.11.20
     ├── accuracy_summary.json
     ├── perf_harness_summary.json
     └── <system>
        └── <benchmark>
            └── <scenario>
                ├── mlperf_log_accuracy.json
                ├── mlperf_log_detail.txt
                ├── mlperf_log_summary.txt
                └── mlperf_log_trace.json
```

The checker then had several issues.
First, it was unable to check the loadgen files as they were not in the exact `run_1, power, ranging` folders. 
The second is that the additional files (`*_summary.json`) were causing the checker to fail.
The third problem is that modifying the file structure (pulling the loadgen into the correct folders) is not allowed by the checker.

I propose the ability to use optional flags to tell the checker where the `ranging` and `run_1` log folders are within the main `session_directory` folder. Either this or have the checker travel subfolders and parse which is which. 

I am not sure, however, how to solve the issue of additional files.

**Comment by ykurlaev** [comment]
This should have been closed with #109 but #200 is also related. @trevor-cockrell can you please verify if it is still an issue in 1.0 and on the latest master branch?

**Comment by trevor-cockrell** [comment]
Confirmed working from the master branch that this will pull parse subfolders for logs.
